src/map/attack.h

```diff
@@ -50,6 +51,15 @@ enum PHYSICAL_ATTACK_DIRECTION
     RIGHTATTACK = 1,
 };
 
+enum ATTACK_ANIMATION
```

Comment:
```cpp
enum class AttackAnimation
{
    RIGHTATTACK = 0,
    LEFTATTACK = 1,
    RIGHTKICK = 2,
    LEFTKICK = 3,
    THROW = 4
};
```

Reply:
and access with `AttackAnimation::RIGHTATTACK` etc

Reply:
Can do on access, why InitCapped enums? We're not doing that anywhere else?

Reply:
cause https://github.com/DarkstarProject/darkstar/blob/9f4baa3c33c19885f45b82f95b7fc26096343b74/src/map/modifier.h#L29

the reason i want this `enum class` is cause it's not the same as the other `ANIMATION` enum, so enum class to avoid ambiguity

Reply:
Very cool, thanks.